index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Style.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <h1>Hello World</h1>
+    <h2>Hello there, we are software developers</h2>
```

Comment:
I think there should be used an id or class.

Reply:
- [ ] It would be a good example which I mentioned above to give a kind code review for students.

Reply:
please use semantic tags and id/class attribute. semantic tags make your code clear and readable. by id/class attribute, you can easily target your HTML codes.